exercises/concept/vehicle-purchase/.docs/hints.md

```diff
@@ -9,14 +9,14 @@
 ## 2. Choose between two potential vehicles to buy
 
 - Use a [relational operator][mdn-relational-operators] to determine which option comes first in dictionary order.
-- Then set the value of a helper variable depending of the outcome of that comparison with the help of an an [if-else statement][mdn-if-statement].
-- Finally construct the recommendation sentence. For that you can use the [addition operator][mdn-addition] to concatenate the two strings.
+- Then set the value of a helper variable depending on the outcome of that comparison with the help of an [if-else statement][mdn-if-statement].
+- Finally, construct the recommendation sentence. For that, you can use the [addition operator][mdn-addition] to concatenate the two strings.
 
 ## 3. Calculate an estimation for the price of a used vehicle
 
 - Start with determining the percentage based on the age of the vehicle. Save it in a helper variable. Use an [if-else if-else statement][mdn-if-statement] as mentioned in the instructions.
-- In the two if conditions use [relational operators][mdn-relational-operators] to compare the age of the car to the threshold values.
-- To calculate the result, apply the percentage to the original price. For example `30% of x` can be calculated by dividing `30` by `100` and multiplying with `x`.
+- In the two if conditions, use [relational operators][mdn-relational-operators] to compare the car age to the threshold values.
```

Comment:
Is "car age" really someone would say?

Reply:
Yes: https://www.linguee.de/deutsch-englisch/search?source=auto&query=car%20age (while it's a dictionary, Linguee is really helpful to search if a certain phrase is actually being used)

Reply:
I would say "car's age". I'm happy to revert though.

Reply:
no, it is ok as you have it now